samples/peripheral/radio_test/prj_usb.conf

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
```

Comment:
Please use the date consistently - although technically, this is not a new configuration (this configuration existed elsewhere), the file is new, introduced for sysbuild. Please align with other build system files in this sample.